src/styles/card.scss

```diff
@@ -0,0 +1,100 @@
+$mainTextColor: #060b35;
+$secondaryTextColor: #616070;
+$whiteColor: #fff;
+$blueAccent: #00acdc;
+
+.card {
+  border: 1px solid #f3f3f3;
+  width: 199px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 0 16px;
+  border-radius: 5px;
+
+  &__img {
+    max-width: 100%;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0 16px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainTextColor;
+  }
+
+  &__code {
+    margin: 4px 16px 16px;
```

Comment:
To avoid specifying `margin: 0 16px;` to all elements just add padding to the parent element